environment

```diff
@@ -128,6 +128,10 @@ azul_vars=(
     # Typically only set for main deployments.
     AZUL_ENABLE_MONITORING
 
+    # Enables logging of all API Gateway traffic through CloudWatch (1 yes, 0 no)
```

Comment:
But add this comment to the the one for AZUL_ENABLE_MONITORING.

Reply:
:+1: